docs/en/query_language/table_functions/index.md

```diff
@@ -1,8 +1,30 @@
 # Table functions
 
-Table functions can be specified in the FROM clause instead of the database and table names.
-Table functions can only be used if 'readonly' is not set.
-Table functions aren't related to other functions.
+Table function is a method of creating a table.
 
+You can use table functions in:
+
+* [CREATE TABLE](../create.md#create-table-query) query.
+
+    It's an alternative method of creating a table.
```

Comment:
this doesn't explain alternative to what + it's as normal as just specifying columns and engine